repos/fdbt-site/src/pages/ticketRepresentation.tsx

```diff
@@ -37,6 +37,26 @@ const getFareTypeDesc = (fareType: TicketType) => {
     }
 };
 
+const getFareTypeHint = (fareType: TicketType) => {
+    switch (fareType) {
+        case 'period':
+            return [
+                'Unlimited travel within a geographical zone',
+                'Unlimited travel on specific service or set of services',
+            ];
```

Comment:
Rather than relying on list ordering, I think we should return objects here, so we can then lookup in the component by the hint's key, rather than by a positional argument

```javascript
return {
    geoZone: 'Unlimited travel within a geographical zone',
    multipleServices: 'Unlimited travel on specific service or set of services',
};
```
```javascript
{
    value: 'geoZone',
    label: 'A ticket within a geographical zone',
    hint: fareTypeHint.geoZone,
}
```

This will make it easier for future changes, like supporting additional `fareTypes` as it's not reliant on ordering, so won't result in accidental reordering of the lists resulting in incorrect messages appearing in the rendered component